ui/src/containers/DeleteBuildButton.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { MenuItem } from 'patternfly-react';
+import { connect } from 'react-redux';
+import { deleteBuildConfig } from '../actions/buildConfigs';
+
+const DeleteBuildButton = (props) => {
+  function triggerDeletion() {
+    props.deleteBuildConfig(props.jobName);
```

Comment:
There is one tiny issue - when the delete is not successful the dropdown menu stays opened. To fix this `document.dispatchEvent(new MouseEvent('click'));` can be added after this line. It seems to be a known bug in react-bootstrap [1].

[1] https://draghici.net/2017/03/10/react-react-bootstrap-menuitem-wont-hide-menu-click/